.travis.yml

```diff
@@ -12,20 +12,42 @@ python:
   # - "3.7" # wait for https://github.com/travis-ci/travis-ci/issues/9815
 
 env:
-  - TOXENV=py-django111
-  - TOXENV=py-django20
-  - TOXENV=py-django21
+  - TOXENV=py-django111-mysql
+  - TOXENV=py-django111-postgres
+  - TOXENV=py-django111-sqlite
+  - TOXENV=py-django20-mysql
+  - TOXENV=py-django20-postgres
+  - TOXENV=py-django20-sqlite
+  - TOXENV=py-django21-mysql
+  - TOXENV=py-django21-postgres
+  - TOXENV=py-django21-sqlite
 
 matrix:
   exclude:
     - python: "2.7"
-      env: TOXENV=py-django20
+      env: TOXENV=py-django20-mysql
     - python: "2.7"
-      env: TOXENV=py-django21
+      env: TOXENV=py-django20-postgres
+    - python: "2.7"
+      env: TOXENV=py-django20-sqlite
+    - python: "2.7"
+      env: TOXENV=py-django21-mysql
+    - python: "2.7"
+      env: TOXENV=py-django21-postgres
+    - python: "2.7"
+      env: TOXENV=py-django21-sqlite
     - python: "3.4"
-      env: TOXENV=py-django21
+      env: TOXENV=py-django21-mysql
+    - python: "3.4"
+      env: TOXENV=py-django21-postgres
+    - python: "3.4"
+      env: TOXENV=py-django21-sqlite
   fast_finish: true
```

Comment:
I really dislike this matrix explosion, but it looks like there's currently no more elegant way to solve this ( travis-ci/travis-ci#1519 ).